core/src/main/resources/error/error-classes.json

```
@@ -1537,6 +1537,12 @@
     ],
     "sqlState" : "42883"
   },
+  "UNRESOLVED_USING_COLUMN_FOR_JOIN" : {
+    "message" : [
+      "USING column <colName> cannot be resolved on the <side> side of the join. The <side>-side columns: [<plan>]."
+    ],
+    "sqlState" : "42877"
```

42877 is for illegally qualified columns e.g. USING (t.c1) (a syntax error for us)
```suggestion
    "sqlState" : "42703"
```

---

Oh... I understand what is the differences. Thanks!!